Tests/MongoSwiftTests/EventLoopBoundMongoClientTests.swift

```diff
@@ -0,0 +1,140 @@
+@testable import MongoSwift
+import Nimble
+import NIO
+import TestsCommon
+
+final class EventLoopBoundMongoClientTests: MongoSwiftTestCase {
```

Comment:
each of these tests need to pass in the `elg` to `withTestClient` for automatic cleanup, since otherwise we'll be leaking resources.

Reply:
oops, yes, I forgot to update these after `withTestClient` was changed.